tests/test_settlements.py

```diff
@@ -0,0 +1,190 @@
+import json
+
+import pytest
+
+from mollie.api.error import IdentifierError
+from mollie.api.objects.settlement import Settlement
+from mollie.api.resources.settlements import Settlements
+from tests.utils import assert_list_object
+
+SETTLEMENT_ID = 'stl_jDk30akdN'
+
+
+def test_settlements_resource_class(client, response):
+    response.get('https://api.mollie.com/v2/settlements', 'settlements_list')
+    client.settlements.list()
```

Comment:
Why are you performing an API call here? The test also succeeds when you don't perform the call.

Reply:
copy leftovers i think